src/main/java/seedu/application/model/job/interview/InterviewType.java

```diff
@@ -0,0 +1,92 @@
+package seedu.application.model.job.interview;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.application.commons.util.AppUtil;
+
+/**
+ * Represents an Interview's type in the application book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidInterviewType(String)}
+ */
+public class InterviewType {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Type is not case sensitive and should only be in the form: \n"
+                    + "BEHAVIOURAL, TECHNICAL, CASE, GROUP, PHONE, VIDEO, ONLINE, ONSITE, OTHER";
+
+    public static final String TO_ADD_INTERVIEW_TYPE = "TO_ADD_INTERVIEW_TYPE";
+    public static final InterviewType EMPTY_INTERVIEW_TYPE = new InterviewType(TO_ADD_INTERVIEW_TYPE);
+    public final String interviewType;
+    /**
+     * Enum for interview types
+     */
+    public enum InterviewTypes {
```

Comment:
This is a good implementation instead of what we considered previously which is splitting these into 2 different fields: `isOnline` + `interviewType`

The current implementation is more logical 👍